.gitignore

```diff
@@ -3,3 +3,5 @@ autom4te.cache
 config.*
 stamp-h1
 libtool
+tags
+cscope.files
```

Comment:
There might be other cscope generated files like `cscope.out`, can we make it `cscope.*`

Reply:
I have pushed a commit that changes it to "cscope.*"